core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4557,7 +4557,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-
+  public static final PropertyKey WORKER_BLOCK_HEARTBEAT_REPORT_CAPACITY_THRESHOLD =
```

Comment:
i feel `capacity` and `threshold` kinda mean the same thing. So maybe `worker.block.heartbeat.threshold`? The word "threshold" implies something will happen when this size is reached.

Reply:
i'd name it to worker.block.heartbeat.report.size.threshold